generated/nidmm/nidmm.proto

```diff
@@ -226,17 +223,12 @@ enum CalibrationType {
   CALIBRATION_TYPE_NIDMM_VAL_EXTERNAL_AREA = 1;
 }
 
-enum CompensatedOhms {
```

Comment:
Need to override OffsetCompensatedOhms to generate as CompensatedOhms. Same enum, different names.